### PR TITLE
Adjust readme to std-readme spec

### PR DESCRIPTION
**Pull request by nmarley**
This is in prep for release.

* Align README w/std-readme spec
* Clean up / reword README a bit
* Add a LICENSE file

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,22 +1,43 @@
-# Dash Network end-to-end tests
+# dash-network-e2e-tests
```
[the remark this answers is not in the document]
Comment (nmarley) [reply README.md]: @shumkov  What do you suggest for short description for this repo?

[the remark this answers is not in the document]
Comment (shumkov) [reply README.md]: > This repo tests the Dash Platform by running some end-to-end tests against a Dash network

?

**Comment by nmarley** [comment]
I don't even understand the status of this at this point or what you'd like to do.

**Comment by shumkov** [comment]
@nmarley You asked me what short description we can use... I would like to merge your changes. I've updated the branch with a new short description.

**Comment by nmarley** [comment]
Ok, cool. Thanks. Want to re-open it then?

File: README.md (hunk 1 above)
[the remark this answers is not in the document]
Comment (nmarley) [reply README.md]: Oh, I think I see... you suggest "This repo tests the Dash Platform by running some end-to-end tests against a Dash network" for a short desc? The way Github formats as markdown confused me.

**Comment by shumkov** [comment]
@

File: README.md (hunk 1 above)
[the remark this answers is not in the document]
Comment (shumkov) [reply README.md]: Yep... I rephrased it a bit: 
"The test suite for end-to-end testing the Dash Platform by running some real-life scenarios against a Dash network"

**Comment by nmarley** [comment]
@shumkov Ready to merge? I can't approve as this is my own PR.

**Pull request merged**